workspace/build_target_test.go

```diff
@@ -0,0 +1,114 @@
+package workspace
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/yourbase/yb/runtime"
+)
+
+func TestBuildTarget_EnvironmentVariables(t *testing.T) {
+	type fields struct {
+		Name        string
+		Tools       []string
+		Commands    []string
+		Environment []string
+	}
+	type args struct {
+		data runtime.RuntimeEnvironmentData
+	}
+	// Sets up dotenv
+	cwd, err := filepath.Abs(".")
+	if err != nil {
+		t.Fatalf("Unable to find current work dir: %v", err)
+	}
+	testPath := filepath.Join(cwd, "TESTING")
```

Comment:
With Go 1.15, you can use [`t.TempDir()`](https://pkg.go.dev/testing?tab=doc#T.TempDir) to procure a directory that gets cleaned up automatically.

Reply:
Man, that's nice!!